src/Widgets/TickListWidget/TickListWidget.scss

```diff
@@ -5,10 +5,18 @@
     display: flex;
     align-items: center;
 
-    .icon {
+    .icon.bi {
       color: var(--bs-primary);
       margin-right: 0.5rem;
       padding: 0.3rem 0;
     }
   }
 }
+
+.bg-primary .tick-list-widget .icon.bi {
+  color: var(--bs-secondary) !important;
```

Comment:
* [ ] This still leaves us with issues: nested colors and different background colors. But if you'd like to move this part to #641, FYI. Otherwise see there for how to style it properly.